devdocs/troubleshooting-guide.md

```diff
@@ -0,0 +1,205 @@
+# Developer Troubleshooting Guide
+
+1. [Troubleshooting project setup](#troubleshooting-project-setup)
+    * [Setup checklist](#setup-checklist)
+    * [Common errors and solutions](#common-errors-and-solutions)
+2. [Troubleshooting test failures](#troubleshooting-test-failures)
+    * [Optimizing IDE layout for testing](#optimizing-ide-layout-for-testing)
+    * [How automated browser tests work](#how-automated-browser-tests-work)
+    * [Getting help to resolve test failures](#getting-help-to-resolve-test-failures)
+    * [Other errors and solutions](#other-errors-and-solutions)
```

Comment:
@wkurniawan07 

I agree with you that it is Common errors and solutions but when I'm tagging them with the hyperlink both are redirecting to the first link

Please kindly suggest a fix for this